rhel-edge-84-virt-test.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# Test script to test RHEL for edge on RHEL 8.4 virtual.
+# Will prepare the env, download upstream test script and run it.
+
+set -uo pipefail
+source /etc/os-release
+
+function greenprint {
+    echo -e "\033[1;32m${1}\033[0m"
+}
+
+# Initialize repo
+greenprint "Configure yum repository"
+sudo tee /etc/yum.repos.d/rhel84.repo << EOF
+[RHEL-8-NIGHTLY-BaseOS]
+name=baseos
+baseurl=http://download-node-02.eng.bos.redhat.com/rhel-8/development/RHEL-8/latest-RHEL-8.4.0/compose/BaseOS/x86_64/os/
+enabled=1
+gpgcheck=0
+[RHEL-8-NIGHTLY-AppStream]
+name=appstream
+baseurl=http://download-node-02.eng.bos.redhat.com/rhel-8/development/RHEL-8/latest-RHEL-8.4.0/compose/AppStream/x86_64/os/
+enabled=1
+gpgcheck=0
+EOF
+
+greenprint "Install packages: git wget ansible httpd osbuild-composer composer-cli"
+sudo dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+sudo dnf install -y --nogpgcheck osbuild-composer composer-cli git curl jq expect wget ansible podman httpd qemu-img qemu-kvm virt-install libvirt-client libvirt-daemon-kvm bash-completion
+sudo rpm -qa|grep osbuild
+
+greenprint "Start image builder service"
+sudo systemctl enable --now osbuild-composer.socket
```

Comment:
this starts osbuild-composer.socket and .service already. So the following two lines of codes are not required.
And you have a lot of osbuild-composer settings changed later, move this to the end of this file.

Reply:
ok.